yarn.lock

```diff
@@ -2160,11 +2160,6 @@
   resolved "https://registry.yarnpkg.com/@sindresorhus/is/-/is-0.7.0.tgz#9a06f4f137ee84d7df0460c1fdb1135ffa6c50fd"
   integrity sha512-ONhaKPIufzzrlNbqtWFFd+jlnemX6lJAgq9ZeiZtS7I1PIf/la7CW4m83rTXRnVnsMbW2k56pGYu7AUFJD9Pow==
 
-"@sindresorhus/is@^2.0.0":
```

Comment:
I think we shouldn't commit any changes to the `yarn.lock` file. Could you undo the changes?

Reply:
I copied the yarn.lock file from develop, which is what the new changes (to 3.3.4) are on the right. This was the recommendation when the nvm upgrade started messing with that file.